pkg/types/compactor.go

```diff
@@ -65,6 +70,7 @@ func (c *CompactorConfig) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&c.FullSnapshotLeaseName, "full-snapshot-lease-name", c.FullSnapshotLeaseName, "full snapshot lease name")
 	fs.StringVar(&c.DeltaSnapshotLeaseName, "delta-snapshot-lease-name", c.DeltaSnapshotLeaseName, "delta snapshot lease name")
 	fs.BoolVar(&c.EnabledLeaseRenewal, "enable-snapshot-lease-renewal", c.EnabledLeaseRenewal, "Allows compactor to renew the full snapshot lease when successfully compacted snapshot is uploaded")
+	fs.DurationVar(&c.SleepFor.Duration, "sleep-timeout-in-seconds", c.SleepFor.Duration, "Timeout for sleep command after the snapshot upload is over")
```

Comment:
```suggestion
	fs.DurationVar(&c.SleepFor.Duration, "sleep-for-seconds", c.SleepFor.Duration, "Sleep duration (in seconds) after compaction is completed")
```

The term `timeout` signifies that some action is to be performed before the specified timeout is elapsed. But here this isn't the case. Rather, this is a `sleep` after the compaction operation is completed, only so that prometheus can have enough time to scrape metrics. hence removed the term `timeout`.

Reply:
Please also ensure that you have addressed the [review comment](https://github.com/gardener/etcd-backup-restore/pull/660#discussion_r1303776914)

Reply:
The field name `SleepFor` is too generic and does not tell anything about the intended use case. It is only there to allow prometheus to scrape metrics. We should then call it `MetricsScrapeWaitDuration` instead or something similar.